Netkan/CKAN-netkan.csproj

```diff
@@ -160,7 +160,6 @@
   </ItemGroup>
   <ItemGroup>
     <EmbeddedResource Include="..\CKAN.schema">
-      <LogicalName>CKAN.NetKAN.CKAN.schema</LogicalName>
```

Comment:
Was there a problem with this `LogicalName` element? It was added to fix something.

Reply:
Likewise, Visual Studio did this automatically. Since it was added to fix an issue I’m the past, I will definitely add it back since there is no need to remove it.

Reply:
This is fixed too. Thanks!